app/controllers/manage_consents_controller.rb

```diff
@@ -40,9 +43,20 @@ def update
     case current_step
     when :confirm
       handle_confirm
-    when :who
+    when :parent_details
       model = @consent.draft_parent
-      handle_who
+      handle_parent_details
+
+      # @consent.draft_parent might not be persisted at this point,
+      # so we need to save using
```

Comment:
I'm not sure I understand this comment 🤔

Reply:
I think I did a not-very-good job of explaining why we can't just rely on `render_wizard` to do the right thing here, like in the other cases:

* if the user is going thru this flow for the first time, `consent.parent_id` is `nil` in the DB and `consent.parent_id` is a new record (set in `set_parent`) so you'd need to call `consent.save` to persist it in addition to saving `parent`
* if the user is going back to edit this page before confirming, `consent.parent_id` is already defined and it would be sufficient to just persist any changes on the `parent` without needing to touch `consent`

Maybe the comment is superfluous?

Reply:
Yeah, I don't think we need the comment, there are some other cases where we can't rely on `render_wizard`